### PR TITLE
Fix AvatarGroup text button overrides

### PR DESCRIPTION
**Pull request by dovilel**

### CODE REVIEW

**Comment by dovilel** [comment]
@jonathanadler Hey. Turns out I really did not do overrides correctly and it worked for me in examples by accident. Fixing, so I can actually override it from where I need to :)

File: src/components/AvatarGroup/AvatarGroup.st.css [hunk 1]
```diff
@@ -156,19 +156,16 @@
...
-       MainTextColor '"--TextButtonColor"',
-       MainTextFont '"--TextButtonFont"'
+       MainTextColor value(TextButtonColor),
+       MainTextFont value(TextButtonFont)
     );
 }
 
-:size(xSmall) .textButton,
-:size(xxSmall) .textButton {
+.smallTextButton {
```
Comment (jonathanadler) [inline src/components/AvatarGroup/AvatarGroup.st.css]: Why add a class and not use the `state`?

Comment (dovilel) [reply src/components/AvatarGroup/AvatarGroup.st.css]: turns out starting with just :size doesn't work on processed css. I added .root now before it, and it's ok. So changed back to original classes, looks clean now.

**Comment by dovilel** [comment]
@jonathanadler please check again! Thanks

**Pull request merged**